app/styles/main.css

```diff
@@ -391,13 +391,19 @@ paper-tab{
   background-color: #333333;
 }
 
-.page__header-title {
+.page__header--title {
```

Comment:
Double dash represents modifier in BEM (Block Element Modifier). Header title and type are elements and they should be called header-title and header-type, with one dash.